src-django/authentication/views.py

```diff
@@ -99,6 +99,7 @@ def login(request):
     form = AuthenticationForm(data=request.POST)
     valid_form = form.is_valid()
     token_key = None
+    user_details = None
```

Comment:
you could init this to `{}` so the return statement doesn't need to check for None